packages/itmat-interface/test/serverTests/users.test.ts

```diff
@@ -1031,6 +1028,8 @@ describe('USERS API', () => {
                 console.warn(chalk.yellow('[[WARNING]]: Skipping test "log in with incorrect totp (user)" because SKIP_EMAIL_TEST is set to "true".'));
                 return;
             }
+            /* Set timeout for the test */
+            jest.setTimeout(30000);
```

Comment:
i think this is still setting global timeout?

set the individual test timeout by:
```javascript
test('testname', () => {}, 30000)
```

Reply:
ok :D thanks